components/constraints.js

```diff
@@ -12,8 +12,11 @@ class Constraints extends Component {
     if (!constraints) return <div />;
     return (
       <ul style={{ listStyle: 'none', paddingLeft: 0 }}>
-        {(constraints.get('default') || constraints.get('default') === 0) &&
-          <li>default value: {constraints.get('default')}</li>
+        {constraints.has('default') &&
+          (constraints.get('type') === 'boolean' ?
+            (constraints.get('default') ? 'default value: true' : 'default value: false') :
```

Comment:
I'm not 100% sure I'm reading this right- maybe it's the cough medicine but I'm finding the nested ?: to be a little challenging.

constraints.get('type') checks the value of the schema's "type" property, right?  Or is it checking  a JavaScript runtime type.  Assuming it's the schema field, what happens if type is set to ["boolean", "null"] or is unset?  A boolean default is still allowed in those cases.

What happens if we just do

{constraints.has('default') && <li>default value: {constraints.get('default')}

If we want to ensure that defaults are valid against their schemas, we should do that elsewhere.